src/next-app/components/Navbar.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import styles from '../styles/components/Navbar.module.css';
+import Button from './Button';
+import ProfileModal from './ProfileModal';
+import { IoMdMenu, IoMdContact } from 'react-icons/io';
+import Link from 'next/link';
+import { NavbarData } from './NavbarData';
+import { IconContext } from 'react-icons';
+
+function Navbar() {
+  const [sidebar, setSidebar] = useState(false);
+
+  const [profile, setProfile] = useState(false);
+
+  const showSidebar = () => setSidebar(!sidebar);
```

Comment:
Function isn't named accurately. Maybe call it toggleSidebar instead? Also, if a setState depends on the previous state, you need to use the anonymous function version of setState (eg: `setSidebar((prevSidebar) => !prevSidebar)`).